projects/playground/src/app/pages/grid/component/grid.component.ts

```diff
@@ -67,6 +72,9 @@ export class GridComponent implements OnInit, OnDestroy, AfterViewInit {
     editedEntity?: any;
     editedIndex?: number;
 
+    scrollableGridStrategy = ResizeStrategy.ScrollableGrid;
+    immediateNeighbourHalt = ResizeStrategy.ImmediateNeighbourHalt;
```

Comment:
Nit: for naming consistency

```suggestion
    immediateNeighbourHaltStrategy = ResizeStrategy.ImmediateNeighbourHalt;
```